README.md

```diff
@@ -47,7 +47,16 @@ Ensure your container environment is running before using these commands.
 
 #### To start the development environment for the first time
 
-Clone the current repository using `git clone https://github.com/WordPress/wordpress-develop.git`. Then in your terminal move to the repository folder `cd wordpress-develop` and run the following commands:
+You can get started using the local development environment with these steps:
+
+1. Go to https://github.com/WordPress/wordpress-develop and fork the repository to your own GitHub account. 
+1. Then clone the forked repository to your computer using `git clone https://github.com/<your-username>/wordpress-develop.git`.
+1. Add the origin repo as an `upstream` remote via `git remote add upstream https://github.com/WordPress/wordpress-develop.git`;
```

Comment:
```suggestion
1. Navigate into the directory for the cloned repository using `cd wordpress-develop`
1. Add the origin repo as an `upstream` remote via `git remote add upstream https://github.com/WordPress/wordpress-develop.git`;
```

Missing step. I believe you'll just get a "fatal: not a git repository" error without it.